frontend/src/containers/NewGroup.js

```diff
@@ -244,6 +249,7 @@ export class NewGroup extends Component {
       mission: attr.mission,
       longDescription: attr.contribute ? `${attr.description}\n\n# Contribute\n\n${attr.contribute}` : attr.description,
       website: attr.website,
+      HostId: hostCollective.settings.HostId,
```

Comment:
where are we setting `isActive` to false when creating this group? Didn't see that on the backend